index.bs

```diff
@@ -1309,15 +1309,66 @@ Verifiable Random Selection Procedure</h5>
 			The incomplete terms are assigned in result order.
 	</ol>
 
+<h5 id="AB-TAG-recall">
+Recalling AB or TAG members</h5>
+
+<h6 id="individual-removal">
+Individual Removal</h6>
+
+	Individual participants of the [=Advisory Board=] or the [=Technical Architecture Group=]
+	can be <dfn export lt="remove|removal">removed</dfn> from those groups
+	if they are found by their peers
+	to be grossly neglecting their duties,
+	or to be acting in a way that seriously hampers the group's ability to function normally.
+
+	A chair of the [=AB=] or [=TAG=] <em class=rfc2119>must</em> hold a hearing
+	on the potential [=removal=] of a participant
+	if requested by at least three of the participants in the group.
+	After giving the individual in question
+	an opportunity to defend themselves,
+	a vote on the proposed [=removal=] is held.
+	If at least three quarters of the participants in the group,
+	excluding the individual who is the subject of such vote,
+	then vote in favor of the proposal,
+	the individual's seat on [=AB=] or [=TAG=] is [=vacated=] immediately.
+
+<h6 id="collective-removal">
+Collective removal</h6>
+
+	The [=Advisory Committee=] <em class=rfc2119>may</em>
+	hold a <dfn export>vote of no confidence</dfn>
+	in the [=Advisory Board=] or the [=Technical Architecture Group=].
+
+	An [=Advisory Committee representative=] initiates a [=vote of no confidence=]
+	by sending a request to the Team, and <em class=rfc2119>should</em> also share this request with the Advisory Committee.
+	The request <em class=rfc2119>must</em> identify whether the [=AB=] or [=TAG=] is targeted,
+	and <em class=rfc2119>should</em> also include the rationale.
+
+	Within one week, the [=Team=] <em class=rfc2119>must</em> initiate an [=Advisory Committee Override=]
+	on the proposal.
+
+	The conclusion of the [=Advisory Committee Override=] is final,
+	and cannot be the subject of a [=Formal Objection=] nor of an [=AC Appeal=]:
+	if the [=Advisory Committee Override=] proposal passes,
+	all seats on [=AB=] or [=TAG=] are [=vacated=] immediately;
+	if it fails, it cannot be invoked on the same body
+	sooner than six months since their previous invocation.
+
 <h5 id="AB-TAG-vacated">
 Elected Groups Vacated Seats</h5>
 
-	An [=Advisory Board=] or [=TAG=] participant's seat is vacated when:
+	An [=Advisory Board=] or [=TAG=] participant's seat is <dfn lt="vacated|vacant">vacated</dfn> when:
 
 	<ul>
 		<li>
 			the participant resigns, or
 
+		<li>
+			the participant is [=removed=], or
+
+		<li>
+			a [=vote of no confidence=] in the body they are part of is successful, or
+
 		<li>
 			an Advisory Board or TAG participant changes affiliations
 			such that the <a href="#AB-TAG-constraints">Advisory Board and TAG participation constraints</a> are no longer met,
```

Comment:
I can only suggest through line 1374, but similar changes should be made through line 1384.
```suggestion
	An [=Advisory Board=] or [=TAG=] participant's seat is <dfn lt="vacated|vacant">vacated</dfn> when any of the following occurs:

	<ul>
		<li>
			The participant resigns.
		</li>
		<li>
			The participant is [=removed=].
		</li>
		<li>
			A [=vote of no confidence=] in the body they are part of is successful.
		</li>
		<li>
			An Advisory Board or TAG participant changes affiliations
			such that the <a href="#AB-TAG-constraints">Advisory Board and TAG participation constraints</a> are no longer met.
		</li>
```

Reply:
@TallTed I'm going to close this discussion, since the text is pre-existing and this is an editorial change to it that is not tied to this PR. Feel free to open a separate PR. :)